### DISCUSSION TITLE
B站动态的评论区内评论字体不渲染

### DISCUSSION THREAD

**Discussion by lelouch1021** [F9y4ng/GreasyFork-Scripts#201] [问答专区(Question / Answer)] [answered]
评论下面的回复就没有问题
我发现在#195中提到PostScript Name 我这个应该不是这个问题吧？我用默认字体也没用
字体可以更改 但是没有渲染效果
![image](https://user-images.githubusercontent.com/23654806/230897093-906a32ea-c477-412a-b2ae-f0e9fd305fa9.png)
具体的设置如下 就是默认的
![image](https://user-images.githubusercontent.com/23654806/230896690-323f1d0f-c937-429a-b538-427dbbce9b43.png)
![image](https://user-images.githubusercontent.com/23654806/230896707-c5e73e29-2afa-4d73-8b2b-20377406c359.png)

**Comment by lelouch1021** [comment]
我发现我退回v2023.03.18.1就没有这个问题了
![image](https://user-images.githubusercontent.com/23654806/230898566-8f42af3c-ef64-4a79-bb38-fddf96d69c05.png)
是因为修改过需要渲染的网页元素吗？在新版本中我应该如何修改？

**Comment by F9y4ng** [comment]
你的意思是评论第一条没有描边阴影效果？ 这个似乎是B站旧版站点上的问题，新版上不存在这个问题，但我记得这个已经被修正过了。

![新版](https://user-images.githubusercontent.com/15855068/230918696-5c87e721-a902-4f34-a6ae-aa2a7f0a3727.png)

从你的截图上看，字体的样式都是正常的，检查一下是否是去广告扩展造成的？

**Comment by F9y4ng** [comment, marked as answer]
Chrome下我没发现问题，Firefox下倒是有这种情况，将**需要渲染的网页元素**内容替换成：
```css
:not(i,head *):not([class*='glyph']):not([class*='icon']):not([class*='fa-']):not([class*='vjs-']):not([class*='mu-'])
```
试试看。

**Comment by F9y4ng** [comment]
我已发布新版本v2023.04.10.2，更新后，重置数据就可使用修正后的过滤规则了。

**Reply by lelouch1021** [reply]
好的 谢谢您
